src/main/java/teammates/common/datatransfer/FeedbackParticipantType.java

```diff
@@ -141,4 +141,25 @@ public String getDisplayNameGiver() {
     public String getDisplayNameRecipient() {
         return displayNameRecipient;
     }
+    
+    /**
+     * @return A user-friendly {@FeedbackParticipantType} list representing participant.
```

Comment:
what do u mean by supplied parameter? .The parameter is fixed (meaning its called by 3 `*Action` classes only) right?. So i think `a list of {@link FeedbackParticipantType} objects ` is only sufficient. So haven't included in the comment.